src/AWS/Lambda/RuntimeClient.hs

```diff
@@ -151,3 +154,16 @@ postInitError' Endpoints{..} session error' = do
   liftIO $ S.post session errorURL error''
   return ()
 
+retry :: (MonadIO m, MonadCatch m) => m a -> m a
+retry = retryWithExponentialBackoff 3 1 100
```

Comment:
Not sure what is the approach with constants we want to have, this is a way to have a "default" retry